src/main/java/com/bina/lrsim/simulator/samples/Samples.java

```diff
@@ -325,7 +329,10 @@ protected final void filterScoreLength(LengthLimits limits) {
 
     for (int idx = 0; idx < lengths.size(); ++idx) {
       final MultiPassSpec spec = new MultiPassSpec(lengths.get(idx));
-      if ( spec.numPasses < limits.minNumPasses || spec.numPasses > limits.maxNumPasses || spec.fragmentLength > limits.maxFragmentLength || spec.fragmentLength < limits.minFragmentLength) {
+      if ( spec.numPasses < limits.minNumPasses ||
```

Comment:
please revert this line to avoid merge conflict, unless there's actual functional changes